### PR TITLE
Expose some necessary fields from Verified ID and request

### PR DESCRIPTION
**Pull request by nithyaganeshng**
Exposed couple of fields from Verified ID and Presentation request which will be useful. Also updated the min and target sdk versions.

**Validation:**
Please include here how it was verified that the PR works as intended.

**Type of change:**
- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [x] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [x] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdPresentationRequest.kt [hunk 1]
```diff
@@ -16,4 +16,6 @@ interface VerifiedIdPresentationRequest : VerifiedIdRequest<Unit> {
 
     // Completes the request and returns nothing if successful.
     override suspend fun complete(): VerifiedIdResult<Unit>
+
+    fun getNonce(): String
```
Comment (logangirvin) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdPresentationRequest.kt]: I don't think this is required; not all request formats have nonces (although they should) so at the least having this return `String?`.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedId.kt [hunk 2]
```diff
@@ -21,7 +21,10 @@ interface VerifiedId {
     val expiresOn: Date?
 
     // Information needed to style a Verified ID.
-    val style: VerifiedIdStyle
+    val style: VerifiedIdStyle?
```
Comment (logangirvin) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedId.kt]: Shouldn't all Verified IDs have a style?

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiableCredential.kt [hunk 3]
```diff
@@ -22,20 +23,18 @@ internal class VerifiableCredential(
...
     override val expiresOn = raw.contents.exp?.let { Date(it * 1000L) }
 
-    val types = raw.contents.vc.type
-
-    override val style = contract.display.toVerifiedIdStyle()
+    override val style = contract?.display?.toVerifiedIdStyle()
 
     override fun getClaims(): ArrayList<VerifiedIdClaim> {
-        val claimDescriptors = contract.display.claims
+        val claimDescriptors = contract?.display?.claims
```
Comment (mauesrog) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiableCredential.kt]: Why is this optional now?

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedIdClaim.kt [hunk 4]
```diff
@@ -9,12 +9,15 @@ package com.microsoft.walletlibrary.verifiedid
  * Represents each claim that is embedded in a VerifiedID.
  */
 data class VerifiedIdClaim(
-    // Represents name of the claim.
+    // Represents id of the claim.
     val id: String,
```
Comment (mauesrog) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedIdClaim.kt:13]: Why do we need to separate label and id?

Comment (logangirvin) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedIdClaim.kt:13]: the `id` for claims is the field name within the credential (ie `givenName`,) while the label is a localized string (`Given Name`.)

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdPresentationRequest.kt (hunk 1 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/VerifiedIdPresentationRequest.kt]: I believe they should. But made it a nullable field.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedId.kt (hunk 2 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedId.kt]: They should. Making contract nullable for a special case affected these. But reverted the changes and moving the special handling outside of Wallet Library.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiableCredential.kt (hunk 3 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiableCredential.kt]: Same reason as above.

**Review by logangirvin — APPROVED** [review]

**Pull request merged**